### DISCUSSION TITLE
The Frida Kahlo Museum is also known as what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#397] [Galaxy Trivia] [answered]
**Hint:** 💙🏠

**Comment by peterbaccaro** [comment]
Frida Kahlo Museum is also known as the **Blue House** (**La Casa Azul** in Spanish) for the structure's cobalt-blue walls.

**Comment by thuongtruong1009** [comment, marked as answer]
The Frida Kahlo Museum, located in Mexico City, is also known as the Casa Azul or the Blue House. This was the house where Frida Kahlo was born and lived most of her life. Today, the house has been turned into a museum, which contains many of Kahlo's personal belongings, as well as artwork and other items related to her life and work.
